### PR TITLE
Feature/update entry format

### LINKED ISSUE #33: Remove feelings archive button from feelings page

### LINKED ISSUE #36: Reformat journal entry header to be a statement using date, feeling, and activity.

### PR DESCRIPTION
**Pull request by KatherineBlaine**
## Files Changed:

- Header.js 
- Journal Entry.js

## Updates:

- Removed feelings archive button from Header
- Updated the header for the journal entry page. Let me know what y'all think of the format!

## Notes:
- The PR template didn't come up when I started this PR? Not sure why just wanted to point that out.

## Closes:
closes #36 
closes #33

### CODE REVIEW

File: client/src/Components/Header/Header.js [hunk 1]
```diff
@@ -13,13 +13,11 @@ const Header = () => {
           <header>
             <nav>
               <Link to='/' className='logo'>BEAM</Link>
-              <Link to='/how-you-felt' className='uni-btn'>Feelings Archive</Link>
```
Comment (jfogiato) [inline client/src/Components/Header/Header.js:16]: GO ON, 'GIT!

Comment (jfogiato) [reply client/src/Components/Header/Header.js:16]: I don't remember exactly, but could we get rid of the `<Switch>` and other logic now that we only have one route in the Nav?

File: client/src/Components/JournalEntry/JournalEntry.js [hunk 2]
```diff
@@ -10,15 +11,14 @@ const JournalEntry = ({ journal }) => {
...
         <div className='log-stats'>
-          <h3>{formattedDate.toLocaleString(DateTime.DATE_MED)}</h3>
-          <h3>I felt {feeling}</h3>
-          <h3>Activity I tried: {activity}</h3>
+          <h3>{`Today you were feeling ${feeling} so you decided to ${activity}`}</h3>
         </div>
         <div className='journal-content'>
           <p className='journal-text'>{journalEntry}</p>
         </div>
+      <Link to={'/how-you-felt'} className="uni-btn">←</Link>
```
Comment (jfogiato) [inline client/src/Components/JournalEntry/JournalEntry.js:21]: `<Links>` - [so hot right now. ](https://media.tenor.com/IH7C6xNbkuoAAAAC/so-hot-right-now-trending.gif)

**Review by jfogiato — APPROVED** [review]
Pulled it down and it looks great! Thanks KB! Appreciate all your work.

Once we add the edit feature to the entry, it will be🤌

**Review by chrissycooper — APPROVED** [review]

File: client/src/Components/Header/Header.js (hunk 1 above)
Comment (jfogiato) [reply client/src/Components/Header/Header.js:16]: Issue added!

File: client/src/Components/JournalEntry/JournalEntry.js [hunk 3]
```diff
@@ -10,15 +11,14 @@ const JournalEntry = ({ journal }) => {
   return (
     <section className='journal-boundary'>
       <div className='journal-entry'>
-        <h2>Journal Entry</h2>
+        <h2>{formattedDate.toLocaleString(DateTime.DATE_MED)}</h2>
         <div className='log-stats'>
-          <h3>{formattedDate.toLocaleString(DateTime.DATE_MED)}</h3>
-          <h3>I felt {feeling}</h3>
-          <h3>Activity I tried: {activity}</h3>
+          <h3>{`Today you were feeling ${feeling} so you decided to ${activity}`}</h3>
```
[the remark this answers is not in the document]
Comment (espressoGoddess) [reply client/src/Components/JournalEntry/JournalEntry.js:16]: I like Joe's idea, and I also love how this is turning out KB!

File: client/src/Components/JournalEntry/JournalEntry.js [hunk 4]
```diff
@@ -10,15 +11,14 @@ const JournalEntry = ({ journal }) => {
   return (
     <section className='journal-boundary'>
       <div className='journal-entry'>
-        <h2>Journal Entry</h2>
+        <h2>{formattedDate.toLocaleString(DateTime.DATE_MED)}</h2>
```
[the remark this answers is not in the document]
Comment (espressoGoddess) [reply client/src/Components/JournalEntry/JournalEntry.js:14]: so much love for that package now that i know how to use it 💩, I am glad we are all getting a chance to use it

File: client/src/Components/JournalEntry/JournalEntry.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (espressoGoddess) [reply client/src/Components/JournalEntry/JournalEntry.js:16]: I like Joe's idea, and I also love how this is turning out KB!

**Review by espressoGoddess — APPROVED** [review]

**Pull request merged**